SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/QuestionsDAO.java

```diff
@@ -0,0 +1,12 @@
+package com.example.smartcarmqttapp;
+
+import androidx.room.Dao;
+import androidx.room.Query;
+
+import java.util.List;
+
+@Dao
+public interface QuestionsDAO {
```

Comment:
@JohnWebb00 Could you maybe explain the exact nature of this class again, the only relevance of this class I see is to query the DB and call method getAll(); returning a list of questions

Reply:
This is my mistake. I forgot to remove the class after we scrapped the idea of using the room database. I will delete the class